lang/en.json

```
@@ -154,6 +154,14 @@
                     "tokenReloadsWeapon": "{token} reloads their {weapon}",
                     "withAmmunition": "with {ammunition}."
                 },
+                "spendAmmoFromMagazine": {
+                    "warningAdvancedAmmunitionNeeded": "PF2e Ranged Combat - Spend Ammo from Magazine can only be used if the Advanced Ammunition System is enabled.",
+                    "warningNpcNotSupported": "PF2e Ranged Combat - Magazine Reload is currently not supported for NPCs.",
```

This should be:
```suggestion
                    "warningNpcNotSupported": "PF2e Ranged Combat - Spend Ammo from Magazine can only be used if the Advanced Ammunition System is enabled for this NPC.",
```

It looks like you copied this from the magazine reload message, which is actually wrong since NPCs CAN reload magazines if they have the Advanced Ammunition System set up.